gui.lua

```diff
@@ -18,7 +18,7 @@ end
 
 local function get_tag_caption(tag)
     str = signal_id_to_rich_text(tag.icon, " [img=utility/custom_tag_in_map_view] ") ..
-                " " .. util.trim(tag.text:sub(config.tag_prefix_len + 1):gsub(util.ASSIGNEE_PATTERN, "[color=blue]@%1[/color]"))
+                " " .. util.trim(tag.text:gsub("TODO:?", ""):gsub(util.ASSIGNEE_PATTERN, "[color=blue]@%1[/color]"))

```

Comment:
I need to read up on my Lua again, but wouldn't that replace `TODO` at any point in the message, not just the start?

Reply:
Ah yeah, that's not good. Luckily was an easy fix.